saml_idp.gemspec

```diff
@@ -41,6 +41,7 @@ defaults in a Production environment. Post any issues you to github.
   s.add_dependency('builder')
   s.add_dependency('httparty')
   s.add_dependency('nokogiri')
+  s.add_dependency('xmlenc', '>= 0.6.4')
```

Comment:
In our case (US Government), encrypting personally identifiable information is always required. I can't speak for the SAML users everywhere, but anyone dealing with PII (credit cards, e.g.) should probably be considering encryption, on top of whatever encrypted transport (SSL) they might be using.